### ISSUE TITLE
Question: Does slngen pass in a property during project evaluation

### ISSUE THREAD

**Issue by ViktorHofer** [microsoft/slngen#331]
Does slngen pass in a property during the project evaluation by which I can identify that it's triggered by slngen?

**Comment by jeffkl** [comment]
Yes, the property `IsSlnGen` is set to `true` during evaluation.  Here are all the global properties it sets:

https://github.com/microsoft/slngen/blob/e9b8452b8300168a0d81ff721c385526af6ff197/src/Microsoft.VisualStudio.SlnGen/ProgramArguments.cs#L297-L300

**Comment by jeffkl** [comment]
Related to #132

**Comment by ViktorHofer** [comment]
Thank you.
